_posts/en/newsletters/2021-02-10-newsletter.md

```diff
@@ -0,0 +1,231 @@
+
+The review club discussion focused on understanding compact blocks, high versus
+low bandwidth modes, and versioning use and compatibility before diving into the
+changes in the code.
+
+{% include functions/details-list.md
+  q0="What are compact blocks?"
+  a0="Compact blocks, specified in [BIP152][], are a way to relay blocks
+     across the Bitcoin peer-to-peer network with less bandwidth use. They take
+     advantage of the fact that during transaction propagation, peers have
+     already seen most of the transactions contained in the received
+     block. Compact blocks can be relayed either in high bandwidth mode or low
+     bandwidth mode. When relayed in high bandwidth mode, compact blocks can
+     also speed up block propagation."
+
+  q1="What is required for compact blocks to work?"
+  a1="The receiver must have a mempool containing transactions that are likely
+     to be included in the block. Compact blocks are therefore only useful for
+     relaying blocks at or near the tip of the block chain. For older blocks,
+     the receiver won't have the transactions in its mempool and would need to
+     request them using a `getblocktxn` message. In such cases, just requesting
+     the full block is more efficient."
+
+  q2="How do compact blocks save bandwidth?"
+
+  a2="Rather than using full transaction ids, compact blocks contain short ids
+     that are smaller in size but sufficient to uniquely identify
+     transactions. A node receiving a compact block matches each short id with a
+     transaction in its mempool to reconstruct the full block. This greatly
+     reduces block relay bandwidth."
+
+  a2link="https://bitcoincore.reviews/20799#l-90"
+
+  q3="What is the difference between \"high bandwidth\" and \"low bandwidth\"
+     modes?"
+
+  a3="In high bandwidth mode, compact blocks are sent unsolicited, trading
+     higher bandwidth use for improved latency, whereas in low bandwidth mode
+     they are sent only on request after an `inv` or `headers` message was
+     received. In high bandwidth mode, compact block messages can be relayed
+     prior to full validation, requiring only that the block header be valid
+     before relay."
+
+  a3link="https://bitcoincore.reviews/20799#l-156"
+
+  q4="How do we choose which peers are in high bandwidth mode?"
+  a4="We select [up to three peers][three] that recently sent us a novel valid
+     block. This logic is in the net processing function,
+     `MaybeSetPeerAsAnnouncingHeaderAndIDs`."
+  a4link="https://bitcoincore.reviews/20799#l-219"
+
+  q5="Why can we stop supporting version 1 compact blocks?"
+
+  a5="BIP152 supports two versions: version 1 (without witnesses) and version 2
+     (with witnesses). Version 2 is required to reconstruct segwit
+     blocks. Segwit was activated in August 2017, so providing version 1
+     pre-segwit compact blocks to peers is no longer useful; without the
+     witnesses, peers are unable to validate the consensus rules on
```

Comment:
"validate the consensus rules on them" sounds awkward to me. "of them", "against them" maybe?